packages/mobile/src/hooks/useLoadOfflineTracks.ts

```diff
@@ -57,9 +57,17 @@ export const useLoadOfflineData = () => {
     }[] = []
     for (const collectionId of offlineCollections) {
       try {
-        dispatch(addCollection(collectionId))
-        if (collectionId === DOWNLOAD_REASON_FAVORITES) continue
+        if (collectionId === DOWNLOAD_REASON_FAVORITES) {
+          dispatch(addCollection({ collectionId, isFavoritesDownload: false }))
```

Comment:
I'm probably just reading the code wrong, but should `isFavoritesDownload` be set to `true` here?

Reply:
or I guess the "Favorites" collection is not considered a Favorites download?